model/build/build.go

```diff
@@ -194,6 +194,18 @@ func (b *Build) SetAborted(aborted bool) error {
 	)
 }
 
+// SetActivated sets the build activated field to the given boolean.
+func (b *Build) SetActivated(activated bool) error {
```

Comment:
Should we be calling SetActivated to true now for when tasks are scheduled, since the builds might not be activated?

Reply:
I think this still needs to be addressed re the todo haha

Reply:
It's done here: https://github.com/evergreen-ci/evergreen/pull/6036/commits/2df525b1032374d19d92137c6880c33f44aaafb7

And I tested it on staging and it's working as expected

Reply:
Ah sounds good; we need to remove the todo then